ci/jenkins/test-mc.sh

```diff
@@ -468,7 +468,10 @@ function collect_coverage {
       kubectl exec -i $mc_controller_pod_name -n ${namespace} ${kubeconfig} -- kill -SIGINT $controller_pid
       cov_dir="${COVERAGE_DIR}/$mc_controller_pod_name-$timestamp"
       mkdir -p $cov_dir
-      kubectl cp ${namespace}/$mc_controller_pod_name:/tmp/coverage/* $cov_dir/ ${kubeconfig}
+      files=(`kubectl exec ${namespace}/$mc_controller_pod_name ${kubeconfig} -- ls /tmp/coverage/`)
```

Comment:
I am not sure this is a valid syntax for `kubectl exec`

```
abasSMD6R:~ abas$ kubectl exec kube-system/antrea-agent-8sspj -- antrea-agent --version
error: the server doesn't have a resource type "kube-system"
abasSMD6R:~ abas$ kubectl exec -n kube-system antrea-agent-8sspj -- antrea-agent --version
antrea-agent version v2.0.0 linux/amd64 go1.21.9
```

Reply:
Fixed it, thanks.